java-backend/src/main/java/org/kframework/backend/java/symbolic/InitializeRewriter.java

```diff
@@ -120,20 +120,17 @@ public void chainOptions() {
 
     @Override
     public synchronized Rewriter apply(org.kframework.definition.Definition def) {
-        Module mainModule = def.mainModule();
-        TermContext initializingContext = TermContext.builder(new GlobalContext(fs, globalOptions, krunOptions,
```

Comment:
The TermContext created here is essentially not used. You see below, it is only used to access GlobalContext, and CounterValue that stayed zero. So this refactoring essentially removes this TermContext and leaves only the GlobalContext.